lib/tpm2.c

```diff
@@ -861,6 +861,53 @@ tool_rc tpm2_tr_set_auth(
     return tool_rc_success;
 }
 
+tool_rc tpm2_activatecredential(
+        ESYS_CONTEXT *esysContext,
+        tpm2_loaded_object *activatehandleobj,
+        tpm2_loaded_object *keyhandleobj,
+        const TPM2B_ID_OBJECT *credentialBlob,
+        const TPM2B_ENCRYPTED_SECRET *secret,
+        TPM2B_DIGEST **certInfo) {
+
+    ESYS_TR keyobj_session_handle = ESYS_TR_NONE;
+    tool_rc tmp_rc = tpm2_auth_util_get_shandle(
```

Comment:
if you never have an rc, you don't need a tmp_rc as it's used elsewhere. When you have a tmp_rc it's usually because you only want to assign to rc on failure and goto a label that returns rc. In short, rename this to rc.

Reply:
fixed